Reference/steering-display/src/main.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+@file    main.cpp
+@brief   Main file for Steering Display Firmware
+@version 1.0
+@date    
+@author  
+*/
+
+#include <Arduino.h>
+#include <SPI.h>
+
+#include "can.hpp"
+#include "cal.hpp"
+
+#include "EVE_target.h"
+#include "EVE_commands.h"
+#include "tft.h"
+
+#define CAN_OE 10
+
+void setup() {
+    Serial2.begin(115200);
+    pinMode(EVE_CS, OUTPUT);
+    digitalWrite(EVE_CS, HIGH);
+    pinMode(EVE_PDN, OUTPUT);
+    digitalWrite(EVE_PDN, LOW);
+
+    pinMode(CAN_OE, OUTPUT);
+	digitalWrite(CAN_OE, LOW);
```

Comment:
Formatting here same as below in the template.

Reply:
indent fix needed. TODO